test/test_helper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "warning"
+
```

Comment:
This is a holdover from when I had to deal with warnings being raised, which I think might've followed from the rebase, but it's unnecessary now. I'll remove it.

```suggestion
```